src/integration/pages/frontPage/user.amp.js

```diff
@@ -0,0 +1,9 @@
+const { amp } = global;
```

Comment:
I made some changes so that we don't need to destructure from global so you can delete this line